service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java

```diff
@@ -722,6 +722,11 @@ public JobProperties exportDevices(String exportBlobContainerUri, Boolean exclud
      * Create a bulk export job.
      *
      * @param exportDevicesParameters A JobProperties object containing input parameters for export Devices job
+     *                                This API also supports identity bases storage authentication, identity authentication
```

Comment:
type: This API also supports identity ~bases~ based storage authenticatio

Reply:
done